components/Chips/examples/ChipsActionExampleViewController.m

```diff
@@ -0,0 +1,116 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "ChipsExamplesSupplemental.h"
+
+#import "MaterialChips.h"
+#import "MDCChipViewColorThemer.h"
+
+@implementation ChipsActionExampleViewController {
+  UICollectionView *_collectionView;
+  MDCChipView *_sizingChip;
+  MDCSemanticColorScheme *_colorScheme;
+}
+
+- (void)loadView {
+  [super loadView];
+
+  _colorScheme = [[MDCSemanticColorScheme alloc] init];
+
+  // Our preferred CollectionView Layout For chips
+  MDCChipCollectionViewFlowLayout *layout = [[MDCChipCollectionViewFlowLayout alloc] init];
+  layout.minimumInteritemSpacing = 10;
+
+
+  // Action chips should allow single selection, collection view default is based on signle
+  // selection. Note that MDCChipCollectionViewCell manages the state of the chip accordingly.
+  _collectionView = [[UICollectionView alloc] initWithFrame:CGRectZero collectionViewLayout:layout];
+
+  // Since there is no scrolling turning off the delaysContentTouches makes the cells respond faster
+  _collectionView.delaysContentTouches = NO;
```

Comment:
Is this because of scrolling or is it required because of chip behavior?

Reply:
It's not required, I double checked. it's just faster response.